examples/pytorch/ogb/ngnn/README.md

```diff
@@ -0,0 +1,387 @@
+# GraphSage/GCN (FULL)
+
+## Introduction
+
+This is an example of implementing [NGNN](https://arxiv.org/abs/2111.11638) for link prediction in DGL.
+
+We use a model-agnostic methodology, namely Network In Graph Neural Network (NGNN), which allows arbitrary GNN models to increase their model capacity.
+
+The script in this folder experiments full-batch GCN/GraphSage (with/without NGNN) on the dataset: ogbl-ddi, ogbl-collab and ogbl-ppa.
+
+## Installation requirements
+```
+ogb>=1.3.3
+torch>=1.11.0
+dgl>=0.8
+```
+
+## Usage
+
+run the scripts in `./scripts` folder, e.g.
+```{.bash}
+scripts/run_ddi.sh
+```
+and you can modify the arguments of the scripts.
+
+## Experiments
+
+We do not fix random seeds at all, and take over 10 runs for all models. All models are trained on a single V100 GPU with 16GB memory.
+
+### ogbl-ddi
+
+#### performance
+
+<table>
+   <tr>
+      <th></th>
+      <th colspan=3 style="text-align: center;">test set</th>
+      <th colspan=3 style="text-align: center;">validation set</th>
+      <th>#parameters</th>
+   </tr>
+   <tr>
+      <td></td>
+      <td>Hits@20</td>
+      <td>Hits@50</td>
+      <td>Hits@100</td>
+      <td>Hits@20</td>
+      <td>Hits@50</td>
+      <td>Hits@100</td>
+      <td></td>
+   </tr>
+   <tr>
+      <td>GCN(ogb)</td>
+      <td>37.07% ± 5.07%</td>
+      <td>\</td>
+      <td>\</td>
+      <td>55.50% ± 2.08%</td>
+      <td>\</td>
+      <td>\</td>
+      <td rowspan=1>1,289,985</td>
+   </tr>
+   <tr>
+      <td>GCN(paper)</td>
+      <td>47.82% ± 5.90%</td>
+      <td>79.56% ± 3.83%</td>
+      <td>87.58% ± 1.33%</td>
+      <td>63.19% ± 0.97%</td>
+      <td>68.31% ± 0.18%</td>
+      <td>70.89% ± 0.12%</td>
+      <td>1,355,777</td>
+   </tr>
+   <tr>
+      <td>GCN+NGNN(paper)</td>
+      <td>48.22% ± 7.00%</td>
+      <td>82.56% ± 4.03%</td>
+      <td>89.48% ± 1.68%</td>
+      <td>65.95% ± 1.16%</td>
+      <td>70.24% ± 0.50%</td>
+      <td>72.54% ± 0.62%</td>
+      <td rowspan=2>1,487,361</td>
+   </tr>
+   <tr>
+      <td>GCN+NGNN(ours; 50runs)</td>
```

Comment:
It suffices to have the numbers for "(ours; 50 runs)" and perhaps "(paper)" if you want. For the rest numbers, people should refer to the paper.